src/ResourceManager/Sql/Commands.Sql/help/Remove-AzureRmSqlDatabaseRestorePoint.md

```diff
@@ -66,6 +66,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PassThru
+{{Fill PassThru Description}}
```

Comment:
@jugeorge please make the description for this "Specifying this will return output if the command was successful."